docassemble/juvenilesealing/data/questions/petitioner-entrypoint-10.yml

```diff
@@ -38,16 +44,24 @@ code: |
     progressor.increment()  # 1
 
   progressor.increment()  # 2
-
+  show_docket_sheets_info = True
   needs_dockets = False
-  # If user doesn't answer yes for docket sheets, show user the docket sheet description
+  # If user doesn't answer yes for docket sheets,
+  # show user why docket sheets are important
   if ( not have_docket_copies_already ):
     progressor.increment()  # 3
     needs_dockets = True
     # If user wants to see docket sheet instructions
-    if ( wants_dockets_first and request_docket_sheets ):
-      progressor.increment()  # 4
-      not_finishing_end  # end
+    # Note 1: Consider pausing progress here (and remember to
+    # reduce total progress steps.)
+    # Note 2: Not sure how this is getting triggered after
```

Comment:
Note 2 shouldn't be relevant anymore, since we're explicitly invalidating `wants_docket_instructions`